src/hdf5_interface.F90

```diff
@@ -787,6 +791,76 @@ subroutine hdf5_read_string(group, name, buffer, length)
   end subroutine hdf5_read_string
 
 !===============================================================================
+! HDF5_WRITE_STRING_1DARRAY writes string data
+!===============================================================================
+
+  subroutine hdf5_write_string_1Darray(group, name, buffer, length, rank)
```

Comment:
Doesn't look like these are complete/used. Did you want to leave them in as stubs? (same deal for the read/write_string_1darray in output_interface)